packages/database/lib/index.ts

```diff
@@ -17,16 +17,16 @@ export class KnexDatabase {
 
     async migrate(): Promise<any> {
         return retry(
-            async () =>
+            async () => {
                 await this.knex.migrate.latest({
                     directory: directory,
                     tableName: '_nango_auth_migrations',
                     schemaName: this.schema()
-                }),
+                });
+            },
             {
                 maxAttempts: 4,
-                delayMs: (attempt) => 500 * attempt,
-                retryIf: () => true
```

Comment:
It's unclear why there was a retry here

Reply:
isn't it to handle race condition when deploying server and multiple instance tries to run the migrations?